lib/noble.js

```diff
@@ -1,190 +1,127 @@
 // Pre-load elliptic as a backup
 const elliptic = require('./elliptic')
-const bitcoinerlab = require('@exodus/bitcoinerlab-secp256k1')
-const noble = require('@noble/secp256k1')
+const necc = require('@noble/secp256k1')
+const hmac = require('@noble/hashes/hmac')
+const sha256 = require('@noble/hashes/sha256')
 
-const errors = {
-  EXPECTED_PRIVATE: 'Expected Private',
-  EXPECTED_POINT: 'Expected Point',
-  EXPECTED_SIGNATURE: 'Expected Signature'
+function _interopNamespaceDefault (e) {
+  const n = Object.create(null)
+  if (e) {
+    Object.keys(e).forEach(function (k) {
+      if (k !== 'default') {
+        const d = Object.getOwnPropertyDescriptor(e, k)
+        Object.defineProperty(n, k, d.get ? d : {
+          enumerable: true,
+          get: function () { return e[k] }
+        })
+      }
+    })
+  }
+  n.default = e
+  return Object.freeze(n)
 }
 
+const noble = _interopNamespaceDefault(necc)
+
+noble.utils.hmacSha256Sync = (key, ...msgs) =>
+  hmac.hmac(sha256.sha256, key, noble.utils.concatBytes(...msgs))
+noble.utils.sha256Sync = (...msgs) => sha256.sha256(noble.utils.concatBytes(...msgs))
```

Comment:
can we leave a comment that this is the official recommendation: https://github.com/paulmillr/noble-secp256k1?tab=readme-ov-file#usage ?